pkgs/tools/text/nkf/default.nix

```diff
@@ -0,0 +1,20 @@
+{ stdenv, fetchurl }:
+
+stdenv.mkDerivation rec {
+  name = "nkf-${version}";
+  version = "2.1.3";
+
+  src = fetchurl {
+    url = "mirror://sourceforgejp/nkf/59912/${name}.tar.gz";
+    sha256 = "8cb430ae69a1ad58b522eb4927b337b5b420bbaeb69df255919019dc64b72fc2";
+  };
+
+  makeFlags = "prefix=\${out}";
+
+  meta = {
+    description = "tool for converting encoding of Japanese text";
```

Comment:
Please capitalize the first character (see http://nixos.org/nixpkgs/manual/#idm140737319319584).

You may also add a `meta.platforms` attribute.